README.md

```diff
@@ -1,14 +1,49 @@
 # Experimental Synthetics Agent
 
 Synthetic Monitoring with Real Browsers.
-**Please note this is an unsupported experimental project. Expect things to break and change!** 
+**Please note this is an unsupported experimental project. Expect things to break and change!**
 
-Docs are forthcoming, feel free to browse the repository, but we ask that you refrain from asking questions about use till docs
-are published.
+## Usage
 
-If you'd like to stay informed about synthetic monitoring development please visit [our synthetic monitoring page](https://www.elastic.co/what-is/synthetic-monitoring) where you can sign up to be notified of our initial release.
+Install the package
 
-## CI
+```sh
+npm install -g @elastic/synthetics
+```
 
-The CI will not run builds for collaborators PRs that are not approval by a members of Elastic org,
-Elastic Users can launch the CI on PRs by putting a comment like `/test` or manually on the CI.
\ No newline at end of file
+Run the suites via CLI
+
+```
+npx synthetics [options] [dir] [files] file
```

Comment:
I think we should avoid `synthetics` since users will confuse this with the `synthetics` NPM package published by newrelic. Can we replace this with either `elastic-synthetics` or `@elastic/synthetics`

Reply:
Yes good catch.